src/wazuh_db/wdb.h

```diff
@@ -1656,13 +1656,14 @@ int wdb_global_reset_agents_connection(wdb_t *wdb);
 /**
  * @brief Function to get the id of every agent with a specific connection_status.
  *
- * @param wdb The Global struct database.
- * @param status Connection status of the agents requested.
- * @retval JSON with every agent ID on success.
- * @retval NULL on error.
+ * @param [in] wdb The Global struct database.
+ * @param [in] last_agent_id ID where to start querying.
+ * @param [in] connection_status Connection status of the agents requested.
+ * @param [out] output A buffer where the response is written. Must be de-allocated by the caller.
+ * @return wdbc_result to represent if all agents has being obtained or any error occurred.
  */
+wdbc_result wdb_global_get_agents_by_connection_status (wdb_t *wdb, int last_agent_id, const char* connection_status, char **output);
```

Comment:
We can add here a description of the chunks logic. As in other similar methods.

Reply:
The comment was updated !